Taskfile.yml

```diff
@@ -25,6 +25,14 @@ tasks:
       - task: lint
       - task: test
 
+  docs:
+    desc: Generate docs
+    aliases: ["doc"]
+    cmds:
```

Comment:
NOTE: v0.13.0 is fairly old and testing the newest version didn't work seamlessly. Sticking with this version for now.